string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,76 @@
+/* exported titleCase */
+// Convert the inputted string to lower case
+// Create a new array of the inputted title
+// Go through each word in the array and capitalize the first letter of each word.
+// If the word is Javascript, make it JavaScript. Replace the old word with the correct word in the array.
+// If the word is Api, make it API . Replace the old word with the correct word in the array.
+// if the word is not the first word in the title but contains an 'insignificant' word, make it lower case
+// and replace the old word with the new word in the array.
+// If the word contains a dash, capitalize the letter after the dash and add the new word to the array by replacing the old word
+// If there's a colon in the string, capitalize the word after the colon, regardless of what it is. Replace the old word with the new word in the array
+// Concatenate the array into a string with a space between each letter.
+// Return the new string from the function.
+
+function titleCase(title) {
+  var titleLowerCase = title.toLowerCase();
+  var titleArray = titleLowerCase.split(' ');
+
+  for (var i = 0; i < titleArray.length; i++) {
+    var word = titleArray[i];
+    var firstChar = word[0];
+    firstChar = firstChar.toUpperCase();
+    var remainingLetters = word.slice(1);
+    remainingLetters = remainingLetters.toLowerCase();
+    var capWord = firstChar + remainingLetters;
+    titleArray.splice(i, 1, capWord);
+
+    if (titleArray[i].includes('Javascript')) {
+      var jsArray = titleArray[i].split('');
+      for (var a = 0; a < jsArray.length; a++) {
+        if (jsArray[a] === 's') {
+          jsArray[a] = 'S';
+        }
+      }
+      var jsString = jsArray.join('');
```

Comment:
You don't need to do the work of manually assembling this special case, you can just write `"JavaScript" directly into your code as the value to use.